### PR TITLE
feat: use the modal container provided by dapps

### PR DESCRIPTION
**Pull request by nicosantangelo**
Depends on https://github.com/decentraland/decentraland-dapps/pull/35

To test: `npm install --save decentraland-dapps@3.14.0-rc6`

### CODE REVIEW

**Comment by cazala** [comment]
`decentraland-dapps@3.14.0` release 🚀

**Comment by nicosantangelo** [comment]
Updated!

**Comment by belohlavek** [comment]
npm starting and testing

**Comment by belohlavek** [comment]
screen shot 2019-02-20 at 10 29 28

File: src/components/Modals/EditProjectModal/EditProjectModal.tsx [hunk 1]
```diff
@@ -28,45 +28,35 @@ export default class EditProjectModal extends React.PureComponent<Props, State>
...
   }
 
   render() {
-    const { modal, grounds } = this.props
+    const { grounds, name, onClose } = this.props
     const { title, description, selectedGround } = this.state
 
     return (
-      <Modal open={modal.open} className="ProjectDetailsModal" size="small" onClose={this.handleOnClose}>
+      <Modal name={name}>
```
Comment (belohlavek) [inline src/components/Modals/EditProjectModal/EditProjectModal.tsx]: why remove `className="ProjectDetailsModal"`?

Comment (cazala) [reply src/components/Modals/EditProjectModal/EditProjectModal.tsx]: `name` is passed down as `className` by the Modal container

**Comment by nicosantangelo** [comment]
> Before
> screen shot 2019-02-20 at 10 31 46
> 
> After
> screen shot 2019-02-20 at 10 29 28

woot taking a look

**Comment by nicosantangelo** [comment]
Oh! the class of the Modal was different to it's name, fixed @belohlavek

**Pull request merged**